From 050962fb45e426c1226c49a37405f86e85472774 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 18 Jul 2018 20:41:01 -0500
Subject: [PATCH] Remove LKG SDK download for source-build.

This is unused by source-build (because we turn off tests) and fails
in the offline configuration.

This patch can be removed after https://github.com/dotnet/toolset/pull/13
is merged.
---
 build/DotnetCoreSdkLKG.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DotnetCoreSdkLKG.props b/build/DotnetCoreSdkLKG.props
index 5db7c9d39..f4b8e89d3 100644
--- a/build/DotnetCoreSdkLKG.props
+++ b/build/DotnetCoreSdkLKG.props
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <_DownloadAndExtractItem Include="DotNetCoreSdkLKGArchive"
-                           Condition="!Exists('$(DotNetCoreSdkLKGArchive)')">
+                           Condition="!Exists('$(DotNetCoreSdkLKGArchive)') and '$(DotNetBuildFromSource)' != 'true'">
       <Url>$(DotNetCoreSdkLKGDownloadUrl)</Url>
       <DownloadFileName>$(DotNetCoreSdkLKGArchive)</DownloadFileName>
       <ExtractDestination>$(DotNetCoreSdkLKGPublishDirectory)</ExtractDestination>
-- 
2.14.1

